src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/MinimumWageCalculator.cs

```diff
@@ -6,36 +6,32 @@ public class MinimumWageCalculator : IMinimumWageCalculator
     {
         private const decimal WeeksPerMonth = 4m;
         private const decimal WeeksPerYear = 52m;
-        private const string MissingMinWageErrorMessage = "MinWage can't be null.";
         private const string HoursPerWeekZeroErrorMessage = "HoursPerWeek must be greater than 0.";
         private const string IncorrectWageTypeErrorMessage = "WageUnit must be either 'Weekly', 'Monthly' or 'Annually'.";
 
-        public decimal CalculateMinimumWage(CreateApprenticeshipRequest request)
+        public decimal CalculateMinimumWage(decimal wageValue, WageUnit wageUnit, decimal hoursPerWeek)
```

Comment:
Suggest we rename this to `CalculateHourlyMinimumWage` so it reads the intent better.

Reply:
Changed Interface and class to HourlyWageCalculator and method to Calculate